### ISSUE TITLE
API. Create Transaction AuthCApture

### ISSUE THREAD

**Issue by tig4647** [AuthorizeNet/sdk-dotnet#69]
Every call into this using an existing customer profile returns an e0027 error in production. Using older SDK, things are fine and in sandbox everything is fine. The message says test transaction failed. I am setting the environment to product and all other calls work fine. The customer is created, the payment profile is created as well. It seems to be only when I try to charge a profile using the API as documented.
IT seems that in most of this SDK that charging a CIM customer is failing at some point. The old methods do not work an longer and new methods fail in production. I would greatly appreciate some feed back as this will cause us business loss.

**Comment by brianmc** [comment]
We'll look into this as a matter of urgency, thanks for the update.

**Comment by tig4647** [comment]
I wanted to follow up and advise that the Api was fine up to friday at 1400hrs eastern. I am checking my code as well but I dont have any changes for friday.

**Comment by tig4647** [comment]
I am going to close this. I found the issue and recommend documentation is updated for CIM. When creating a payment profile, you MUST select a verification  mode. In my case i had removed this and this caused the new charge to be a test instead of an actual charge. To me the verification would be at the moment you create the payment profile not when you are charging the profile. I have found that most providers do not allow this test. I was able to complete a charge against CIM when i created the profile with verification of "none". I do think that the default functionality should be none but that is an opinion. For sure the verification should NOT happen when I can a transaction request.
